conda/api.py

```diff
@@ -4,11 +4,10 @@
 from .common.constants import NULL
 from .core.package_cache_data import PackageCacheData as _PackageCacheData
 from .core.prefix_data import PrefixData as _PrefixData
-from .core.solve import Solver as _Solver, DepsModifier as _DepsModifier
+from .core.solve import DepsModifier as _DepsModifier, Solver as _Solver
```

Comment:
Why do we do 

```
from .core.solve import DepsModifier as _DepsModifier
```

and then

```
DepsModifier = _DepsModifier
```

Reply:
Yeah we don't need that.

Reply:
Actually, did that to avoid the reassignment error thing.

Reply:
Yeah but I mean why not import it directly as 

`from .core.solve import DepsModifier` ?

Reply:
@kalefranz?

Reply:
Because then you need a line

    DepsModifier = DepsModifier